app/src/main/java/io/intrepid/contest/screens/contestoverview/ContestOverviewActivity.java

```diff
@@ -45,10 +66,27 @@ public void showContestName(String contestName) {
     }
 
     @Override
+    public void showContestDescription(String description) {
+        descriptionTextView.setText(description);
+    }
+
+    @Override
+    public void showCategories(List<Category> categories) {
+        categoryAdapter.setCategories(categories);
+        categoriesRecyclerView.setAdapter(categoryAdapter);
```

Comment:
yea we can change the adapter's data and call notifyDataSetChanged() when we need to. That works for me. I'll make the changes!